### PR TITLE
PS-8830 Document new open_tables_with_triggers status variable (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   docs/ps-variables.md

### CODE REVIEW

**Review by alina-derkach-oaza — APPROVED** [review]
LGTM

File: docs/flexibility/trigger-updates.md [hunk 1]
```diff
@@ -56,6 +58,7 @@ The following status variables are available:
 
 | Variable name                         | Description |
 |---------------------------------------|-------------|
+| `Open_tables_with_triggers` | The current number of TABLE instances with fully-loaded triggers in the Table cache. |
```
Comment (dlenev) [inline docs/flexibility/trigger-updates.md]: Perhaps it is better to be consistent with description of other status variables in this table and say "... in the `table_open_cache`." instead ?

**Review by dlenev — CHANGES_REQUESTED** [review]
Hello Patrick!

I have only one minor suggestion about this patch. Otherwise it looks good to me.

**Review by dlenev — APPROVED** [review]
LGTM.

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**